packages/istanbul-lib-instrument/src/instrumenter.js

```diff
@@ -80,17 +82,18 @@ class Instrumenter {
         }
         filename = filename || String(new Date().getTime()) + '.js';
         const opts = this.opts;
+        const defaultPlugins = [
```

Comment:
I'd prefer that `new Instrumenter({plugins: someList})` replace the default plugins.  The way I envision this:
1. Export `defaultOpts` (in src/index.js as well).
2. Move this defaultPlugins array to the `plugins` property inside `defaultOpts`.
3. If the caller wants to append default plugins they can import defaultOpts and use destructuring or Array.concat to setup opts.plugins.

I'm concerned that someone might need plugins in a specific order or may need to remove a plugin, the current method would not allow this.